core/src/main/java/nl/nn/adapterframework/core/IPullingTriggerListener.java

```diff
@@ -0,0 +1,24 @@
+/*
+   Copyright 2020 Nationale-Nederlanden
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+package nl.nn.adapterframework.core;
+
```

Comment:
Kan je hier in de javadoc het verhaal / de conclusies van de discussie met Eric opnemen?